### PR TITLE
Wales and Scotland name change

### PR DESCRIPTION
**Pull request by gknisely**
# Issue

Wales and Scotland had a name change.  We need to come up with a better design and handle name changes.  Not sure what to do yet.  Anyhow #fixes https://github.com/valhalla/valhalla/issues/3135  Added a small test showing how just one user inputted access can shut off country access overrides.

## Tasklist

- [x] Add tests
- [x] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: lua/admin.lua [hunk 1]
```diff
@@ -64,7 +64,7 @@ drive_on_right = {
 ["South Africa"] = "false",
 ["Sri Lanka"] = "false",
 ["Suriname"] = "false",
-["Scotland"] = "false",
+["Alba / Scotland"] = "false",
```
Comment (kevinkreiser) [inline lua/admin.lua:67]: doesnt this mean that someone converting an older planet will also get screwed? i think the compatible fix would be to include both variants right? just support both for a time. its a bit of baggage to carry but its better than getting bug reports :D

File: lua/admin.lua [hunk 2]
```diff
@@ -191,7 +191,7 @@ function rels_proc (kv, nokeys)
            end
          end
        end
-       if kv["name"] == "England" or kv["name"] == "Scotland" or kv["name"] == "Wales" or kv["name"] == "Northern Ireland" then
+       if kv["name"] == "England" or kv["name"] == "Alba / Scotland" or kv["name"] == "Cymru / Wales" or kv["name"] == "Northern Ireland" then
```
Comment (gknisely) [inline lua/admin.lua:194]: > doesnt this mean that someone converting an older planet will also get screwed? i think the compatible fix would be to include both variants right? just support both for a time. its a bit of baggage to carry but its better than getting bug reports :D

@kevinkreiser Yeah I had the following, but was concerned folks would complain.  Thoughts?

```
if kv["name"] == "Alba / Scotland" then
   kv["name"] = "Scotland"
end

if kv["name"] == "Cymru / Wales" then
   kv["name"] = "Wales"
end
```

Comment (nilsnolde) [reply lua/admin.lua:194]: IMO fine fix:)

**Review by nilsnolde — DISMISSED** [review]

**Comment by nilsnolde** [comment]
urgh.. windows fails with some vcpkg idiocy.. can someone pls re-trigger that workflow, hoping it's just a hiccup? lint build is the usual coverage fail

File: lua/admin.lua (hunk 2 above)
Comment (kevinkreiser) [reply lua/admin.lua:194]: I had actually meant just leave both checks in and use whichever we find. Which means doing it at the other places where you have changes as well. Basically support both names all the way through the code

Comment (kevinkreiser) [reply lua/admin.lua:194]: Yep I know that's what I'm suggesting

Comment (kevinkreiser) [reply lua/admin.lua:194]: Do we know when those relations changed? If it was a long while ago then just ignore my comment. If it was recent then it's a pretty huge gotcha to have to know to use the bleeding edge changeset

Comment (gknisely) [reply lua/admin.lua:194]: @kevinkreiser Scotland was 8 months ago and Wales I am not sure as it keeps timing out on me.

Comment (kevinkreiser) [reply lua/admin.lua:194]: ok cool then we can just ship it as is. we'll assume wales was also a while ago. sorry for the noise i just didnt want to knowingly put a gotcha in there

**Review by kevinkreiser — DISMISSED** [review]

File: lua/admin.lua (hunk 2 above)
Comment (gknisely) [reply lua/admin.lua:194]: @kevinkreiser np.  thanks for the feedback

**Review by kevinkreiser — APPROVED** [review]

**Pull request merged**